README.md

```diff
@@ -1,2 +1,39 @@
+<img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
+
 # openmrs-ocl-client
-OCL client for OpenMRS
+
+> OCL client for OpenMRS is a web service that will allow OpenMRS users to manage their concepts on the web,
+> allow reusability of the existing concepts and allow one to create their concepts if they are not in exsitance.
+
+## Build
+### Requirements
+
+- Node and Npm [Here](https://nodejs.org/en/download/).
+
+### Installation Process
+```
+- Fork the repository https://github.com/openmrs/openmrs-ocl-client#fork-destination-box
```

Comment:
What url should one fork?

Reply:
I copied the URL we are redirected to when we click the fork button but I have discovered that the only alternative is to click the fork button let me rectify that